pkg/controllers/openshift-apiserver.go

```diff
@@ -196,7 +196,7 @@ apiServerArguments:
 	if cfg.AuditLogDir != "" {
 		data = append(data, `
 auditConfig:
-  auditFilePath: `+cfg.AuditLogDir+`openshift-apiserver-audit.log
+  auditFilePath: "`+cfg.AuditLogDir+`/openshift-apiserver-audit.log"
```

Comment:
Will the need for that slash depend on whether the AuditLogDir variable ends in one? Should we use filepath.Join()?